### ISSUE TITLE
Problem when changing the number of anchor boxes

### ISSUE THREAD

**Issue by LucasVandroux** [experiencor/basic-yolo-keras#158]
As long as I keep 5 anchor boxes, I have no problem to train the network but as soon as I add or remove some anchors, the network is not working anymore and I get the following error (here trying with 4 anchors):
```
Traceback (most recent call last):
  File "train.py", line 143, in <module>
    _main_(args)
  File "train.py", line 139, in _main_
    debug              = config['train']['debug'])
  File "/home/lucas/basic-yolo-keras/frontend.py", line 385, in train
    self.model.compile(loss=self.custom_loss, optimizer=optimizer)
  File "/usr/local/lib/python2.7/dist-packages/keras/engine/training.py", line 827, in compile
    sample_weight, mask)
  File "/usr/local/lib/python2.7/dist-packages/keras/engine/training.py", line 426, in weighted
    score_array = fn(y_true, y_pred)
  File "/home/lucas/basic-yolo-keras/frontend.py", line 107, in custom_loss
    pred_box_wh = tf.exp(y_pred[..., 2:4]) * np.reshape(self.anchors, [1,1,1,self.nb_box,2])
  File "/usr/local/lib/python2.7/dist-packages/numpy/core/fromnumeric.py", line 257, in reshape
    return _wrapfunc(a, 'reshape', newshape, order=order)
  File "/usr/local/lib/python2.7/dist-packages/numpy/core/fromnumeric.py", line 62, in _wrapfunc
    return _wrapit(obj, method, *args, **kwds)
  File "/usr/local/lib/python2.7/dist-packages/numpy/core/fromnumeric.py", line 42, in _wrapit
    result = getattr(asarray(obj), method)(*args, **kwds)
ValueError: cannot reshape array of size 8 into shape (1,1,1,5,2)
```

It seems like `nb_box` is set arbitrary at 5 in [`frontend.py` line 27](https://github.com/experiencor/basic-yolo-keras/blob/9b0ef52943a07d7b4e30007f1dfa93eae257c88b/frontend.py#L27).

**Comment by LucasVandroux** [comment]
This problem is already resolved in the pull request #141 but hasn't been merged with the main branch yet.

**Comment by experiencor** [comment]
@LucasVandroux Sorry for the late reply as I was on vacation. Thanks for the letting me know. I will merge it to the main branch in a couple of days.
